trackintel/preprocessing/triplegs.py

```diff
@@ -132,7 +132,9 @@ def _seperate_ids(row):
     trips.drop(columns=["type", "sp_tpls_id"], inplace=True)  # make space so no overlap with activity "sp_tpls_id"
     # Inserting `gaps` and `user_change` into the dataframe creates buffers that catch shifted
     # "staypoint_id" and "trip_id" from corrupting staypoints/trips.
-    trips_with_act = pd.concat((trips, sp_tpls_only_act, gaps, user_change), axis=0, ignore_index=True)
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", message="CRS not set for some of the concatenation inputs.*")
+        trips_with_act = pd.concat((trips, sp_tpls_only_act, gaps, user_change), axis=0, ignore_index=True)
```

Comment:
Is this also related to empty dataframes?

Reply:
I think all dfs are not empty in our test cases. And this case is more complicated :)
This is the initial place that I tried to find what is causing the warning `CRS not set for some of the concatenation inputs.*`, and I still do not know how it is triggered :). The warning persisted when I set all dfs as pandas dataframe, so I guess a df with geom column (with crs) is still considered a gdf for some functions.

I simply silence the warning here, as this warning is only for information purposes, and I think we should not substantially change the code to fix it.